### PR TITLE
Remove link pointing to Keith Peters' Twitter

### LINKED ISSUE #28336: The hyperlink to under the text "Keith Peters" redirects to his inactive twitter account.
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/JavaScript

### What specific section or headline is this issue about?

See also

### What information was incorrect, unhelpful, or incomplete?

The hyperlink redirects to the inactive twitter handle of the Keith Peters. He has moved his social handle to mstdn.social

### What did you expect to see?

I wish this link to redirect to his new mstdn.social handle or point to his blog repository. Please have a look at the screenshot attached to get the accurate info. I can open a PR to fix this issue.

Screenshot 2023-08-02 095950

### Do you have any supporting links, references, or citations?

His new social profile link is https://mstdn.social/@bit101
His blog repository is placed at https://www.bit-101.com/blog/

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/javascript`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/index.md
* Last commit: https://github.com/mdn/content/commit/751d58669499de0c6ea0d5b356e0e1448418c5d3
* Document last modified: 2023-06-30T06:21:35.000Z

</details>

### PR DESCRIPTION
**Pull request by meetanoopverma**
This PR is intended to replace the outdated link pointing to the inactive profile of the Keith Peters with the new updated link.

To redirect every reader to the location where they can find more relevant resources to learn.

It redirects them to get connected with the person concerned and engage in a learning pursuit.

### Related issues and pull requests

"Fixes #28336"
"Relates to #28336"

### CODE REVIEW

**Comment by meetanoopverma** [comment]
Kindly review and merge this PR.

**Review by wbamberg — DISMISSED** [review]
Thanks for your PR! The suggestion from @hamishwillee in https://github.com/mdn/content/issues/28336#issuecomment-1664865407 was to remove the name and link.

**Review by meetanoopverma — COMMENTED** [review]
As per the suggestion, changes have been effected.

File: files/en-us/learn/javascript/index.md [hunk 1]
```diff
@@ -68,4 +68,4 @@ This topic contains the following modules, in a suggested order for working thro
 - [Learn JavaScript](https://learnjavascript.online/)
   - : An excellent resource for aspiring web developers — Learn JavaScript in an interactive environment, with short lessons and interactive tests, guided by automated assessment. The first 40 lessons are free.
 - [Coding math](https://www.youtube.com/user/codingmath)
-  - : An excellent series of video tutorials to teach the math you need to understand to be an effective programmer, by [Keith Peters](https://twitter.com/bit101).
+  - : An excellent series of video tutorials to teach the math you need to understand to be an effective programmer.
```
Comment (bsmth) [inline files/en-us/learn/javascript/index.md:71]:
```suggestion
  - : An excellent series of video tutorials to teach the math you need to understand to be an effective programmer, by [Keith Peters](https://www.bit-101.com/blog/about-me/).
```

**Comment by bsmth** [comment]
Thanks @meetanoopverma - I've added a link to https://www.bit-101.com/blog/about-me/ which contains links to socials / other resources.

**Review by bsmth — APPROVED** [review]

**Pull request merged**